CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+2.16.0 Release notes (YYYY-MM-DD)
+=============================================================
+
+### Bug fixes
+* Including user's schema without overriding the permission schema added for query based Realm [#2017](https://github.com/realm/realm-js/issues/2017).
```

Comment:
Do you know when the bug was introduced? If you know, please add it as `since v...`.